packages/content-management/table_list/src/components/use_tag_filter_panel.tsx

```diff
@@ -53,21 +64,31 @@ export const useTagFilterPanel = ({
   const [isInUse, setIsInUse] = useState(false);
   const [options, setOptions] = useState<TagOptionItem[]>([]);
   const [tagSelection, setTagSelection] = useState<TagSelection>({});
-  const totalActiveFilters = Object.keys(tagSelection).length;
+  const totalActiveFilters = useMemo(
+    () => getTotalActiveFilters(options, fixedTagReferences),
```

Comment:
Why do we pass `options` and not `tagSelection` ? I would also pass an array if `fixedTagReferences` is undefined or null to make `getTotalActiveFilters` simpler

```ts
const getTotalActiveFilters = (
  tagSelection: TagSelection,
  fixedTagReferences: TagReference[]
) =>
  Math.max(
    fixedTagReferences.length,
    Object.keys(tagSelection).length
  );

const totalActiveFilters = useMemo(
  () => getTotalActiveFilters(tagSelection, fixedTagReferences ?? []),
  [fixedTagReferences, tagSelection]
);
...


```

Reply:
It could really just be this

```ts
const totalActiveFilters = useMemo(() => (
    Math.max((fixedTagReferences ?? []).length, Object.keys(tagSelection).length)
  )),
  [fixedTagReferences, tagSelection]
);
...

Reply:
<img width="1704" alt="Screenshot 2023-05-23 at 15 30 45" src="https://github.com/elastic/kibana/assets/6295984/57df1d0e-c154-42f9-a4f9-68527f77a426">